server/views/pages/explore.njk

```diff
@@ -75,5 +75,9 @@
     </div>
   </div>
 
+  <div data-component-name="Tweet block" data-component-id="explore">
```

Comment:
I'm still a little confused about what `name` vs `id` is used for?
`name` for tracking, id for ?